spec/source_file_finder.rb

```diff
@@ -0,0 +1,27 @@
+class SourceFileFinder
+  ALLOWED_FILE_EXTENSIONS = [".md"]
```

Comment:
There's some SCSS and JS files in `source/` that I presumed we want to ignore - let me know if that assumption is incorrect.